docs/pages/workflow/ios-simulator.md

```diff
@@ -8,25 +8,31 @@ import { Terminal } from '~/ui/components/Snippet';
 
 ## Step 1: Install Xcode
 
-This step is very easy but it takes a while. Open up the Mac App Store, search for Xcode, and hit install (or update if you have it already). If you're not able to update because your operating system is out of date, we recommend updating your operating system to the latest version and then updating Xcode. You may run into issues further down the line if your Xcode version is out of date, for example you may not be able to submit your app to the App Store.
+This step is very easy but it takes a while. Open up the Mac App Store, search for [Xcode](https://apps.apple.com/us/app/xcode/id497799835), and hit install (or update if you have it already). 
+If you're not able to update because your operating system is out of date, we recommend updating your operating system to the latest version and then updating Xcode. 
+You may run into issues further down the line if your Xcode version is out of date, for example you may not be able to submit your app to the App Store.
 
 ## Step 2: Install Xcode Command Line Tools
 
-Open Xcode, then choose "Preferences..." from the Xcode menu (or press <kbd>Cmd ⌘</kbd> + <kbd>,</kbd>). Go to the Locations panel and install the tools by selecting the most recent version in the Command Line Tools dropdown.
+Open Xcode, then choose **Preferences...** from the Xcode menu (or press <kbd>Cmd ⌘</kbd> + <kbd>,</kbd>). Go to the Locations panel and install the tools by selecting the most recent version in the Command Line Tools dropdown.
 
-<ImageSpotlight alt="Xcode preferences" src="/static/images/xcode-command-line.png" containerStyle={{ paddingBottom: 0 }} />
+<ImageSpotlight alt="Xcode preferences" src="/static/images/xcode-command-line.png" />
 
 ## Step 3: Try it out
 
-Run your app with `npx expo start` and press <kbd>I</kbd> from the command line. You may get a warning about needing to accept the Xcode license. Run the command that it suggests. Open your app again, success! Or no? If no, please seek help on StackOverflow, Google, or the [Expo CLI section of the forums](https://forums.expo.dev/c/expo-cli). The troubleshooting tips below may be helpful too.
+Run your app with `npx expo start` and press <kbd>I</kbd> from the command line. 
+
+You may get a warning about needing to accept the Xcode license. Run the command that it suggests. Open your app again, success! Or no? 
+If no, please seek help on StackOverflow, Google, or the [Expo Development Tools section of the forums](https://forums.expo.dev/c/expo-dev-tools). 
+The troubleshooting tips below may be helpful too.
 
 <Video file="open-in-ios-simulator.mp4" />
 
-> Pro Tip: Press shift+I in the CLI UI to interactively select a simulator to open.
+> Pro Tip: Press <kbd>Shift</kbd> + <kbd>I</kbd> in the CLI UI to interactively select a simulator to open.
```

Comment:
Just a thought, should we use lowecase characters for `i`? The reason I'm saying this is because by keeping them lower case, these shortcuts represent the same command as mentioned in the "help" of Expo CLI.